tailwind.config.ts

```
@@ -2,13 +2,69 @@ import { type Config } from "tailwindcss";
+                    '4': 'hsl(var(--chart-4))',
+                    '5': 'hsl(var(--chart-5))'
+                },
+                backgroundImage: {
+                    hero_overlay: "url('/hero_overlay.png')",
+                },
+            }
+        }
     },
-  },
-  plugins: [],
+    plugins: [require("tailwindcss-animate")],
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)